packages/x-charts/src/BarChart/BarPlot.tsx

```diff
@@ -288,7 +309,6 @@ function BarPlot(props: BarPlotProps) {
     leave: leaveStyle,
     enter: enterStyle,
     update: enterStyle,
-    immediate: skipAnimation,
```

Comment:
not sure why

Reply:
Me neither... was a mistake on my part, but I don't remember doing it lol